src/UXCapture.js

```diff
@@ -8,35 +9,49 @@ let _onMeasure;
 let _view;
 
 const UXCapture = {
+	_clearMarksAndMeasures: () => {
+		window.performance.clearMarks();
+		window.performance.clearMeasures();
+	},
+
 	/**
 	 * Sets `onMark` and `onMeasure` callbacks on UXCapture singleton
+	 * Also resets the view (supposed to be called once per page anyway)
 	 *
 	 * @param {object} config
 	 */
-	create: (config) => {
+	create: config => {
 		_onMark = config.onMark || NOOP;
 		_onMeasure = config.onMeasure || NOOP;
+		_view = null;
 	},
 
 	/**
 	 * Creates a new View instance.
 	 *
 	 * @param {object} zoneConfigs
 	 */
-	startView: (zoneConfigs) => {
-		_view = new View({
+	startView: zoneConfigs => {
+		const viewProps = {
 			onMark: _onMark,
 			onMeasure: _onMeasure,
 			zoneConfigs,
-		});
+		};
+
+		// if we never had views before, it means we are in page view mode
+		if (_view) {
+			_view = new InteractiveView(viewProps);
+		} else {
+			_view = new PageView(viewProps);
+		}
 	},
 
 	/**
 	 * Updates current view instance with new zones & marks
 	 *
 	 * @param {object} zoneConfigs
 	 */
-	updateView: (zoneConfigs) => {
+	updateView: zoneConfigs => {
```

Comment:
Do you have a prettier or eslint plugin in your editor that is removing this parentheses? If so, could you disable for this repo?

Reply:
prettier does this by default, and that behavior is active in other meetup repos - i suspect the existing parens snuck through before prettier was being applied to this repo

Reply:
Yes, that's just prettier (not sure if it's part of eslint integration though).

@sadafie, any reason you want this to be removed? seems like a normal formatting rule to me, at least I got used to it by now ;)

Reply:
Hmmm, I definitely had prettier committed prior to this branch, but, let's see what happens the next time we commit. I just didn't want a situation where we keep adding and removing across PRs. I'll keep a look out on my end.